packages/price-exporter/package.json

```diff
@@ -21,8 +21,31 @@
     "email": "williams.omayuku@commercetools.de"
   },
   "main": "lib/index.js",
+  "bin": {
+    "price-exporter": "bin/price-exporter.js"
+  },
   "files": [
     "bin",
     "lib"
-  ]
+  ],
+  "dependencies": {
+    "@commercetools/api-request-builder": "^3.1.1",
+    "@commercetools/get-credentials": "^2.1.0",
+    "@commercetools/sdk-client": "^1.4.1",
+    "@commercetools/sdk-middleware-auth": "^3.1.0",
+    "@commercetools/sdk-middleware-http": "^2.1.1",
+    "@commercetools/sdk-middleware-user-agent": "^1.2.0",
+    "JSONStream": "^1.3.1",
+    "bluebird": "^3.5.0",
```

Comment:
My opinion is that it really doesn't make any sense to range your deps if you already are relying on greenkeeper to keep them up to date.
One issue we suffered with range, is that sometimes deps break with some breaking change sneaking in